index.js

```diff
@@ -24,6 +26,12 @@ function vine_parser (url) {
   return match && match[1].length === 11 ? match[1] : url;
 }
 
+var prezi_regex = /^https:\/\/prezi.com\/(.+?)\//;
```

Comment:
If someone neglects to add the trailing slash, will this match? Example: `https://prezi.com/fugumlghvxoy`